FRAuthenticator/FRAuthenticator/Model/Notification/PushNotification.swift

```diff
@@ -240,40 +240,40 @@ public class PushNotification: NSObject, NSSecureCoding, Codable {
     ///   - result: Boolean indicator whether or not PushNotification authentication is approved or denied
```

Comment:
Since the parameter name was changed, please update the name in the inline docs as well (i.e. "result" -> "approved")